legion/cli/legion/cli/parsers/security.py

```diff
@@ -79,10 +90,13 @@ def login(args):
     :type args: argparse.Namespace
     :return: None
     """
-    _check_credentials(args)
-    _save_credentials(args.edi, args.token)
+    if _check_credentials(args):
+        if args.token:
+            _save_temporary_token(args.edi, args.token)
 
-    LOG.info('Success! Credentials were saved.')
+            LOG.info('Success! Temporary token has been saved.')
+        else:
+            LOG.info('You has been authorized before.')
```

Comment:
Let's save EDI URL to the config if login finished successfully.

Reply:
it is being saved...